openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -483,6 +488,52 @@ const TeamDetailsV1 = ({
     return `Are you sure you want to ${text}`;
   };
 
+  const openGroupIcon = useMemo(() => {
+    if (currentTeam.isJoinable) {
+      return <SVGIcons alt="Delete" icon={Icons.OPEN_LOCK} />;
+    } else {
+      return <SVGIcons alt="Delete" icon={Icons.CLOSED_LOCK} />;
+    }
+  }, [currentTeam.isJoinable]);
+
+  const extraDropdownContent: ItemType[] = useMemo(
+    () => [
+      {
+        label: (
+          <Space
+            className="tw-cursor-pointer manage-button"
+            size={8}
+            onClick={handleOpenToJoinToggle}>
+            {openGroupIcon}
+            <div className="tw-text-left open-group" data-testid="open-group">
+              <Row className="tw-mb-1" justify="space-between">
+                <Col>
+                  <p className="tw-font-medium" data-testid="open-group-label">
+                    Open Group
+                  </p>
+                </Col>
+                <Col>
+                  <Switch
```

Comment:
Here we are disabling the switch if the user does not have permission, so can we add the tooltip here?

Reply:
We are not disabling the switch, we are removing the manage dropdown itself. I will remove the disable prop as it is not necessary now.